python/spear/game_world_service.py

```diff
@@ -14,3 +14,75 @@ def open_level(self, level_name):
 
     def set_game_paused(self, paused):
         self._rpc_client.call("game_world_service.set_game_paused", paused)
+
+    def find_actors(self):
+        return self._rpc_client.call("game_world_service.find_actors")
+
+    def find_actors_as_map(self):
+        return self._rpc_client.call("game_world_service.find_actors_as_map")    
+
+    def get_components(self, actor):
+        return self._rpc_client.call("game_world_service.get_components", actor)
+
+    def get_components_as_map(self, actor):
+        return self._rpc_client.call("game_world_service.get_components_as_map", actor)
+
+    def get_children_components(self, actor, include_all_descendants):
+        return self._rpc_client.call("game_world_service.get_children_components", actor, include_all_descendants)
+
+    def get_children_components_as_map(self, actor, include_all_descendants):
+        return self._rpc_client.call("game_world_service.get_children_components_as_map", actor, include_all_descendants)
+
+    def get_object_properties_as_string_from_object(self, object):
```

Comment:
Note that I'm changing some variable names here to match `Unreal.h`

```
    def get_object_properties_as_string_from_uobject(self, uobject):
        return self._rpc_client.call("game_world_service.get_object_properties_as_string_from_uobject", uobject)

    def get_object_properties_as_string_from_ustruct(self, value_ptr, uobject):
        return self._rpc_client.call("game_world_service.get_object_properties_as_string_from_ustruct", value_ptr, ustruct)

    def set_object_properties_from_string_for_uobject(self, uobject, string):
        self._rpc_client.call("game_world_service.set_object_properties_from_string_for_uobject", uobject, string)

    def set_object_properties_from_string_for_ustruct(self, value_ptr, ustruct, string):
        return self._rpc_client.call("game_world_service.set_object_properties_from_string_for_ustruct", value_ptr, ustruct, string)

    def find_property_by_name_on_uobject(self, uobject):
        return self._rpc_client.call("game_world_service.find_property_by_name_on_uobject", uobject, name)

    def find_property_by_name_on_uobject(self, value_ptr, uobject, name):
        return self._rpc_client.call("game_world_service.find_property_by_name_on_ustruct", value_ptr, ustruct, name)
```

Reply:
Slight typo suggestion above
```
    def find_property_by_name_on_uobject(self, uobject, name):
        return self._rpc_client.call("game_world_service.find_property_by_name_on_uobject", uobject, name)
```